test/mocked/User.js

```diff
@@ -0,0 +1,34 @@
+var mockery = require('mockery');
```

Comment:
Do we need this file (`test/mocked/User.js`) still? It seems like this was part of the old tests.